.kitchen.dokken.yml

```diff
@@ -18,6 +18,9 @@ platforms:
   driver:
     image: dokken/amazonlinux
     pid_one_command: /sbin/init
+    intermediate_instructions:
```

Comment:
Do we need this if we're removing the amazon tests?

Reply:
I am hoping that they sort out their issues with DNS. Then we should be able to start using them again.